crypto/x509/x509_set.c

```diff
@@ -279,7 +279,10 @@ static int x509_sig_info_init(X509_SIG_INFO *siginf, const X509_ALGOR *alg,
             ERR_raise(ERR_LIB_X509, X509_R_ERROR_GETTING_MD_BY_NID);
             return 0;
         }
-        siginf->secbits = EVP_MD_get_size(md) * 4;
+        md_size = EVP_MD_get_size(md);
```

Comment:
After we've looked up the digest with `EVP_get_digestbynid()`, can `EVP_MD_get_size()` still return an error?

Reply:
It cannot, but I do not think it matters much. We could eventually do a proper fetch there and that could return some broken digest.